frontend/src/components/tabs/RuntimeNodeDetailsV2.tsx

```diff
@@ -233,6 +257,42 @@ function getTaskDetailsFields(
   return details;
 }
 
+function getNodeVolumeMounts(
+  layers: string[],
+  templateString?: string,
+  element?: FlowElement<FlowElementDataBase> | null,
+): Array<KeyValue<string>> {
+  if (!templateString || !element) {
+    return [];
+  }
+
+  const taskKey = getTaskKeyFromNodeKey(element.id);
+  const pipelineSpec = convertYamlToV2PipelineSpec(templateString);
+  const componentSpec = getComponentSpec(pipelineSpec, layers, taskKey);
+  const platformSpec = convertYamlToPlatformSpec(templateString);
+
+  if (!platformSpec) {
+    return [];
+  }
+
+  // Currently support kubernetes platform
+  const k8sDeploymentSpec = PlatformDeploymentConfig.fromJSON(
+    platformSpec.platforms['kubernetes'].deploymentSpec,
+  );
+  const matchedExecutorObj = Object.entries(k8sDeploymentSpec.executors).find(
+    ([executorName]) => executorName === componentSpec?.executorLabel,
```

Comment:
How does `k8sDeploymentSpec.executors` represent a component with name `executorName` that is in the sub-DAG?

Reply:
Created a internal issue to follow up this item :)